lib/apaka/packaging/deb_package-default.yml

```diff
@@ -59,8 +59,8 @@ distributions:
         labels: 11,sid,default
 # what distribution should be build with which architecture
 architectures:
-    amd64: trusty,xenial,jessie,stretch,buster,bionic
-    i386:  trusty,xenial,jessie,stretch,buster,bionic
+    amd64: trusty,xenial,jessie,stretch,buster,bionic, focal
+    i386:  trusty,xenial,jessie,stretch,buster,bionic, focal
```

Comment:
no spaces between list items so far, so I suggest to remove the space (or add after each comma)

Reply:
I removed spaces